src/components/blocks/ReviewBlock.jsx

```diff
@@ -52,8 +52,25 @@ const ReviewBlock = ({ t, review, shouldLimitLines, linkTo, pageFrom }) => {
     e.stopPropagation();
     e.preventDefault();
 
-    // eslint-disable-next-line no-alert
-    alert(t('ui.message.reportUnderDevelopment', { contact: CONTACT }));
+    const parmas = {
```

Comment:
오타인 것 같아서 수정하면 좋을 것 같습니다! line 68도 같이요 👀 
```suggestion
    const params = {
```